apps/web/src/app/[locale]/challenge/_components/comments/comment.tsx

```diff
@@ -75,6 +75,8 @@ const commentReportSchema = z
 
 export type CommentReportSchemaType = z.infer<typeof commentReportSchema>;
 
+const REPLIES_PAGESIZE = 5;
+
```

Comment:
Adjusted replies to show just 5 at a time, this matches LeetCode

For example, https://leetcode.com/problems/two-sum/description/comments/1565865 
<img width="350" alt="image" src="https://github.com/typehero/typehero/assets/7817695/7b1d94c5-f84a-4f5c-a258-c7f45f09bfea">